ts/features/payments/checkout/analytics/index.ts

```diff
@@ -96,6 +96,8 @@ export const getPaymentAnalyticsEventFromRequestFailure = (
     case "PAYMENT_GENERIC_ERROR_AFTER_USER_CANCELLATION":
       return "PAYMENT_GENERIC_ERROR_AFTER_USER_CANCELLATION";
     case "PAYMENT_VERIFY_GENERIC_ERROR":
+    case "PAYMENT_METHOD_NOT_AVAILABLE_ERROR":
```

Comment:
Are we sure that the error to dispatch on mixpanel is related to the `PAYMENT_METHOD_` and not at the PSP?

Reply:
My bad that was unintentional